solr/contrib/jaegertracer-configurator/README.txt

```diff
@@ -0,0 +1,30 @@
+Welcome to Apache Solr Prometheus Exporter
+========
+
+Apache Solr Jaeger Tracer Configurator (solr-jaeger-tracer) provides a way for you to expose Solr's tracing to Jaeger.
+
+# Setup Jaeger Tracer Configurator
+
+Note that all library of solr-jaeger-tracer must be included in the classpath of all nodes then Jaeger tracer can be setup in `solr.xml` like this:
+
+```
+<tracerConfig name="tracerConfig" class="org.apache.solr.jaeger.JaegerTracerConfigurator">
+    <double name="samplePercentage">100.0</double>
```

Comment:
The default in the example shouldn't be set to 100%. Users might blindly copy the example and pay the price of tracing each request unknowingly. Perhaps set it to 5% to start?

Reply:
On latest PR, `samplePercentage` is 1% if clusterProperty does not override this value.